zenpacklib.py

```diff
@@ -4997,8 +4997,9 @@ def format_message(e):
     def yaml_warning(loader, e):
         # Given a MarkedYAMLError exception, either log or raise
         # the error, depending on the 'fatal' argument.
-
-        print format_message(e)
+        pass
+        # commenting out for 1.1 release
+        # print format_message(e)
```

Comment:
Thanks for merging ... yes there are options like that, but since this is an "internal-only" release for shipping with OpenStack and VSphere I thought this the most expedient thing to do.

ZPL 2.0 will have more fine-grained event logging as well.